storage/object/how-to/use-bucket-website.mdx

```diff
@@ -24,75 +24,33 @@ The Bucket Website feature allows you to host [static websites](https://en.wikip
   - You have [created an Object Storage bucket](/storage/object/how-to/create-a-bucket/)
 </Message>
 
-<Message type="important">
-  To use the bucket website feature, you need to enable it on the bucket or buckets you wish to host your website on.
-</Message>
-
 1. Click **Object Storage** in the **Storage** section of the side menu. The list of your buckets displays.
 2. Click the name of the bucket you wish to host your website on.
-    <Lightbox src="scaleway-object-storage-dashboard.webp" alt="" />
-3. Scroll down to the bottom of the **Bucket settings** tab and click **Enable bucket website**. A pop-up appears.
-    <Lightbox src="scaleway-enable-bucket-website.webp" alt="" />
-4. Click the <Icon name="toggle" /> icon to enable the feature.
-5. Enter the names of the **index** and **error** files of your website that will be uploaded to the bucket.
-    <Lightbox src="scaleway-website-settings.webp" alt="" />
-    The **index** and **error** files are commonly named `index.html` and `error.html`, respectively. You may have already uploaded these files to your bucket, or you can upload them later. Either way, their names must be specified now.
-
-    <Message type="tip">
-      - The index file is the default or home page for the website. It serves as an index to the other pages of the site.
-      - The error file defines the page that displays in case of an error, for example if a user tries to access a page of your website that does not exist.
-
-    Here is an example of the contents of a basic `index.html` file:
-
-      ```html
-      <!doctype html>
-      <html>
-        <head>
-          <title>My Website</title>
-        </head>
-        <body>
-          <div id="main">
-              <h1>Hi! Welcome to my website.</h1>
-          </div>
-        </body>
-      </html>
-      ```
 
-    Here is an example of the contents of a basic `error.html` file:
+3. Click **Enable bucket website** in the **bucket settings** tab. A pop-up appears.
 
-      ```html
```

Comment:
Will we put this info (and for the index file) somewhere else, maybe at the end of the document in a tip box? It might help out users who want to quickly test the feature before using their own complex content?